src/main/kotlin/net/irisshaders/lilybot/extensions/config/Config.kt

```diff
@@ -45,9 +48,28 @@ class Config : Extension() {
 				check {
 					anyGuild()
 					hasPermission(Permission.ManageGuild)
+					requireBotPermissions(Permission.SendMessages, Permission.EmbedLinks)
 				}
 
 				action {
+					val actionLogChannel = guild!!.getChannelOf<TextChannel>(arguments.modActionLog.id)
+
+					val actionLogPermissions =
+						actionLogChannel.getEffectivePermissions(this@ephemeralSlashCommand.kord.selfId)
+
+					if (!actionLogPermissions.contains(Permissions(Permission.SendMessages, Permission.EmbedLinks))) {
+						respond {
+							embed {
```

Comment:
This is literally the only place in the bot we respond with an embed, why?

Reply:
Yes